vnext/Microsoft.ReactNative/Fabric/Composition/CompositionEventHandler.cpp

```diff
@@ -234,14 +236,18 @@ void CompositionEventHandler::HandleIncomingPointerEvent(
     auto componentView = *itComponentView;
     bool shouldEmitEvent = componentView != nullptr &&
         (hasParentEnterListener ||
-         IsViewListeningToEvent(componentView, facebook::react::ViewEvents::Offset::PointerEnter));
+         IsViewListeningToEvent(componentView, facebook::react::ViewEvents::Offset::PointerEnter) ||
+         IsViewListeningToEvent(componentView, facebook::react::ViewEvents::Offset::MouseEnter));
 
     if (shouldEmitEvent &&
         std::find(currentlyHoveredViews.begin(), currentlyHoveredViews.end(), componentView) ==
             currentlyHoveredViews.end()) {
       facebook::react::SharedTouchEventEmitter eventEmitter = componentView->touchEventEmitter();
       if (eventEmitter) {
         eventEmitter->onPointerEnter(event);
+        if (IsMousePointerEvent(event)) {
+          eventEmitter->onMouseEnter(event);
```

Comment:
Not sure where you're handling pointer events for inline spans, but you won't have an IComponentView corresponding to the span that was actually targeted for an event. However, the AttributedString from the state object has a read-only copy of the shadow view for the TextShadowNode corresponding to a fragment, so you can look up the correct event emitter from AttributedString.

Reply:
Thanks for the heads up.  With the latest changes, we now get onPress events properly happening on inline spans.